scripts/build.actions.js

```diff
@@ -44,21 +44,27 @@ class BuildActions extends BaseScript {
       const outPath = path.join(this.config.actions.dist, `${name}.zip`)
       const actionFileStats = fs.lstatSync(actionPath)
 
-      if (!actionFileStats.isDirectory() && !actionFileStats.isFile()) throw new Error(`${action.function} is not a valid file or directory`)
+      if (!actionFileStats.isDirectory() && !actionFileStats.isFile()) {
+        throw new Error(`${action.function} is not a valid file or directory`)
+      }
 
       if (actionFileStats.isDirectory()) {
         // make sure package.json exists
         const packageJsonPath = path.join(actionPath, 'package.json')
         if (!fs.existsSync(packageJsonPath)) {
```

Comment:
minor comment: maybe we could change `utils.getActionEntryFile` so that it returns `'index.js'` if `packageJsonPath` doesn't exist, this would simplify the logic a bit.

Alternatively in `missing required ${this._relApp(packageJsonPath)} for folder actions` would be good to say `"or  missing index.js"`